javaslang-benchmark/pom.xml

```diff
@@ -42,6 +42,11 @@
             <artifactId>jmh-generator-annprocess</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Now that `PriorityQueue` benchmarks are merged in, this may be problematic, as we have `ScalaZ` also :/
Could you maybe use its `Scala` dependency, as I did?